Objects/unicodeobject.c

```diff
@@ -5084,8 +5099,7 @@ find_first_nonascii(const unsigned char *start, const unsigned char *end)
 #if PY_LITTLE_ENDIAN && HAVE_CTZ
         if (p < p2) {
 #if defined(_M_AMD64) || defined(_M_IX86) || defined(__x86_64__) || defined(__i386__)
-            // x86 and amd64 are little endian and can load unaligned memory.
-            size_t u = *(const size_t*)p & ASCII_CHAR_MASK;
+            size_t u = load_unaligned_x86_amd64(p) & ASCII_CHAR_MASK;
```

Comment:
The standard compliant way of loading unaligned memory is to use `memcpy`. If you give `memcpy` a fixed size, then on the platforms which support non-aligned loads (including x86, arm64), you just get a normal load operation:

https://gcc.godbolt.org/z/4fG7h3349

Reply:
Yes, please do it.